pkg/core/storage/memory_store.go

```diff
@@ -83,6 +83,28 @@ func (s *MemoryStore) Batch() Batch {
 	}
 }
 
+// Persist flushes all the MemoryStore contents into the (supposedly) persistent
+// store provided via parameter.
+func (s *MemoryStore) Persist(ps Store) (int, error) {
+	s.Lock()
+	defer s.Unlock()
+	batch := ps.Batch()
+	keys := 0
```

Comment:
dunno if keys is the correct variable name since it looks more as counter of persisted values

Reply:
We're about key-value pairs here, so we can count keys and we can count values and it's gonna be the same in the end.

Reply:
true. I'm talking more about naming. Don't have super strong opinion for that. Just think that naming things as what they actually do is a good way to make code-reading easier for some external reader=)